test/resource/wasm/rust/build.rs

```diff
@@ -0,0 +1,8 @@
+// The default stack size in Rust is 1MB, which causes oversized allocation warnings,
+// because it's allocated in a single chunk as a part of a Wasm Linear Memory.
+// We change the stack size to 128KB using the RUSTFLAGS environment variable
+// in the command below.
+fn main() {
+    println!("cargo:rustc-link-arg=-zstack-size=131072");
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
With this line, cargo will correctly rebuild the files if the `build.rs` gets changed. However, if we try to compile with e.g. `ninja wasm`, cargo won't be called at all, so the files won't be updated.
To deal with that, I think we just need to add the new `build.rs` to the depencency list in 
https://github.com/scylladb/scylladb/blob/d2e089777bf5cca87a81a6ace7c5d67d37365a08/configure.py#L1844-L1848